scripts/prepare_compliance_data.py

```diff
@@ -8,19 +8,34 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import shutil
 import argparse
 import os
-import sys
+import shutil
 import glob
 import pysam
 import utils
+import tempfile
+import zipfile
 
 
 class ComplianceDataMunger(object):
     def __init__(self, args):
         self.inputDirectory = args.inputDirectory
         self.outputDirectory = args.outputDirectory
+        # If no input directory is specified download from GitHub
+        if not args.inputDirectory:
```

Comment:
Should be `` if args.inputDirectory is not None:``, see https://www.python.org/dev/peps/pep-0008/#programming-recommendations point 2.